base/Dockerfile

```diff
@@ -2,20 +2,20 @@
 ARG FROM
 FROM $FROM
 
-# Build libcap 2.32 targeting armhf
+COPY patch/* /
+
+# Build libcap 2.66 targeting armhf
 RUN export DEBIAN_FRONTEND=noninteractive \
     # Install build dependencies
     && apt-get update -y \
     && apt-get install -y --no-install-recommends \
         git \
     # Build libcap
     && cd /root \
-    && git clone git://git.kernel.org/pub/scm/libs/libcap/libcap.git \
+    && git clone --depth 1 --branch libcap-2.66 git://git.kernel.org/pub/scm/libs/libcap/libcap.git \
     && cd libcap \
-    && git checkout 13227f9b2f2b2f222e8022e19bd46db6f6f898c6 \
```

Comment:
If you used commits as hashes, I can add checks that branch we cloned matches commit. It is easier to read, if specifying tag instead of just commit SHA

Reply:
We use commit hashes because they don't change. Tags should be fine though as people don't always move them.

The goal to to make this a reproducible build. So tags would work for that.